R/tm_g_association.R

```diff
@@ -359,22 +359,45 @@ srv_tm_g_association <- function(id,
         )
       })
 
+      # helper function to format variable name
+      foo <- function(x) {
```

Comment:
```suggestion
      format_labels_helper <- function(x) {
```
(and changes below of course)

Best to be more descriptive than just "foo"

Reply:
will do